### PR TITLE
Add new topic  "Further Information "

### PR DESCRIPTION
**Pull request by moinmoin-sh**
@sumnerboy12 here is a proposal for an additional topic that could be added to each FW-doc with links to corresponding information, download, etc.  Some content behind the links is not complete, but should give an idea.

@MakerDockio the pages under guides/getting-started-guides look broken (different than Docs). may be you can have alook?

### CODE REVIEW

File: docs/docs/firmware/state-monitor-esp32.md [hunk 1]
```diff
@@ -211,6 +208,26 @@ The RJ45 pinout for each port is;
...
+
+### Flash Binary
+How to [flash the binary](/guides/getting-started.html#Firmware).
+
+### Home Automation Integration
+examples how to integrate StateMonitor into Home Automation systems
+
+[Home Assistant integration examples](/guides/advanced/home_assistant.html)
+
+[Node Red integration examples](/guides/advanced/node_red.html)
```
Comment (sumnerboy12) [inline docs/docs/firmware/state-monitor-esp32.md:226]: `[Node-RED](https://nodered.org/) integration [examples](/guides/advanced/node_red.html).`

File: docs/guides/advanced/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ tags: [""]
 
 ## Software
  - [Home Assistant](/guides/advanced/home_assistant.html) Sample automations to work with StateMonitor
- - node red
+ - [Node Red](/guides/advanced/node_red.html) Sample flows to work with StateMonitor
```
Comment (sumnerboy12) [inline docs/guides/advanced/README.md:9]: `[Node-RED](/guides/advanced/node_red.html) sample flows to work with [State Monitor](/docs/firmware/state-monitor-esp32.html)`

File: docs/guides/advanced/README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ tags: [""]
 
 ## Software
  - [Home Assistant](/guides/advanced/home_assistant.html) Sample automations to work with StateMonitor
```
Comment (sumnerboy12) [inline docs/guides/advanced/README.md:8]: `[Home Assistant](/guides/advanced/home_assistant.html) sample automations to work with [State Monitor](/docs/firmware/state-monitor-esp32.html)`

File: docs/guides/advanced/node_red.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+# OXRS StateMonitor Node Red integration
+
+
+## Introduction
+This guide shows `automation` samples to be used Node Red. They make use of the event structure that is sent by the StateMonitor via MQTT
+
+For details on `configuration` and `events` see the [StateMonitor](/docs/firmware/state-monitor-esp32.html) documentation.
```
Comment (sumnerboy12) [inline docs/guides/advanced/node_red.md:7]: `State Monitor`

File: docs/guides/advanced/node_red.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+The following samples are using the config topic `conf/58391f` and the state topic `stat/58391f`. Those are the defaults as allocated by the FW using the MAC address of the device. Index numbers need to be replaced according to your set up.
+
+
+
+### Example 1: Use a `switch` to turn light on and off 
+---
+#### Typical use case : standard wall switch. 
+
+to configure input `index: 5`  as `switch` publish :
```
Comment (sumnerboy12) [inline docs/guides/advanced/node_red.md:17]: Capitilise the start of each sentence and ensure each ends with a full stop.

File: docs/guides/advanced/node_red.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
+# OXRS StateMonitor Node Red integration
```
Comment (sumnerboy12) [inline docs/guides/advanced/node_red.md:1]: Wonder if this page should be titled `OXRS Node-RED Integrations` with sub-sections for each firmware type we provide examples for?

Same for your other Home Assistant page?

**Review by sumnerboy12 — CHANGES_REQUESTED** [review]
Think this is a great idea Moin - thanks for getting on with it and doing the work!

**Comment by moinmoin-sh** [comment]
I hope I got the grammar right (don't tell my English teachers) and the integration docs get close to what you suggested

**Review by sumnerboy12 — APPROVED** [review]
Looks great!

**Pull request merged**